ast/src/analyzed/mod.rs

```diff
@@ -161,6 +161,7 @@ impl<T> Analyzed<T> {
             assert!(!to_remove.contains(&poly_id));
             poly.id = replacements[&poly_id].id;
         });
+        //self.defi
```

Comment:
Are we officially a Blockchain product now?

Reply:
removed.